documentjs.json

```diff
@@ -15,10 +15,16 @@
 			},
 			"parent": "Guides",
 			"pageConfig": {"page":"guides"}
+		},
+		"funcunit": {
+			"glob": {
+				"pattern": "funcunit.md"
```

Comment:
I don’t think this file should be necessary, so I would double check that any of this configuration is required.